test/conformance/service_test.go

```diff
@@ -315,15 +315,15 @@ func TestReleaseService(t *testing.T) {
 		expectedThirdRev  = helloWorldText
 	)
 
-	objects, err := test.CreateRunLatestServiceReady(logger, clients, &names, &test.Options{})
+	objects, err := test.CreateReleaseServiceWithLatest(logger, clients, &names, &test.Options{})
```

Comment:
Can we check that this serves the expected text?

Reply:
I've added two checks. We can expand them in future, should we want to validate more fields.